src/libecalc/core/consumers/compressor/component.py

```diff
@@ -49,8 +49,8 @@ def get_max_rate(self, operational_settings: CompressorOperationalSettings) -> L
             operational_settings_this_timestep = operational_settings.get_subset_for_timestep(timestep)
             results.extend(
                 compressor.get_max_standard_rate(
-                    suction_pressures=np.asarray(operational_settings_this_timestep.inlet_pressure.values),
-                    discharge_pressures=np.asarray(operational_settings_this_timestep.outlet_pressure.values),
+                    suction_pressures=np.asarray(operational_settings_this_timestep.inlet_streams[0].pressure.values),
```

Comment:
Again, assuming first stream is the requested. We should probably make this a bit more clear. Maybe requested_inlet_stream should be a separate property than crossover. Need to look at how this works (or should work) in multiple streams also. Should we ask for max_rate per stream?

Reply:
For multiple streams the max standard rate is calculated for each stream. When calculating the max standard rate for one stream, the rates for the other streams are kept constant.